web/src/components/core/InstallButton.jsx

```diff
@@ -23,25 +23,19 @@ import React, { useState } from "react";
 import { useInstallerClient } from "~/context/installer";
 
 import { Button } from "@patternfly/react-core";
-import { Link } from "react-router-dom";
 
 import { If, Popup } from "~/components/core";
 import { _ } from "~/i18n";
 
 const InstallConfirmationPopup = ({ hasIssues, onAccept, onClose }) => {
   const IssuesWarning = () => {
     // TRANSLATORS: the installer reports some errors,
-    // the text in square brackets [] is a clickable link
-    const [msgStart, msgLink, msgEnd] = _("There are some reported issues. \
-Please, check [the list of issues] \
-before proceeding with the installation.").split(/[[\]]/);
 
     return (
       <p>
         <strong>
-          {msgStart}
-          <Link to="/issues">{msgLink}</Link>
-          {msgEnd}
+          { _("There are some reported issues. \
+          Please review any identified concerns in the previous steps before proceeding with the installation.") }
```

Comment:
@imobachgs, If you don't mind, please have a look to the text. They are not my strong skill :P 

Would be nice to have @mvidner and/or @shundhammer, but I wouldn't wait until they are back. We can refine these texts later, if needed.

Reply:
What about replacing "identified concerns" with just "them"? Anyway, we know what we are talking about.

Reply:
Not sure if a user who doesn't speak English very well will understand that. Don't forget that there are _many_ of them. I'd vote to greatly simplify that statement.

It starts with "issues" which has become en vogue in recent years (it wasn't always like that) as an euphemism for "problems". If this is about problems (or potential problems), let's just say it. If not, can we make it clearer what kind of "issues" are meant here?

As for the rest, I am not sure if it's clear to users (even native speakers) what that means, what to do, and what to look for.  I'd prefer something much clearer like "please go back and check the problems outlined in red and make the necessary changes".

We very often (starting with YaST2 in the early 2000s) had the problem of very vague, very generic messages. That typically comes from generalizing some dozen specific messages into one very generic one; generic to the point of being very much useless and just confusing the user.

Reply:
A really nice point, I suppose a more straightforward message is always a better choice.

> "please go back and check the problems outlined in red and make the necessary changes"

Something like this seems fine, but I didn't update it now, waiting for others to jump in.

Reply:
As discussed today in a team daily call, I got what @shundhammer said (thanks a lot for jumping in during your vacation days) but I'm not fully agree on linking the message with the look & feel. Today is red, tomorrow who knows. But more important, there are people that are not able to distinguish colors, reason why color add value only for a group of people.

In any case, we've somehow agreed on looking for a better term and explanation, but in a different issue. So, @balsa-asanovic please keep it as it is for now in order to not make this PR longer than it is already. There are some work to do here too, but we're close to merge it. Leave the message for another PR, once we've it clear.